server/src/server.js

```diff
@@ -570,10 +582,13 @@ app.post("/api/login", function(req, res) {
         deviceId: vars.deviceId,
         userAbTests
       };
+      let accountId = retrieveAccount(vars.deviceId).then((accountId) => {
+        sendParams.accountId = accountId;
+      });
```

Comment:
This looks really racy – if this doesn't complete before `Show.checkOwnership` then none of this will work.

This should be chained with the sendParamsPromise